lvm.py

```diff
@@ -5,28 +5,36 @@
 # Copyright (c) 2016, InnoGames GmbH
 #
 
-from __future__ import print_function
-from __future__ import division
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
 
-from socket import gethostname
-import time, sys
-import subprocess
 
-try:
-    vgdisplay = subprocess.Popen(['/sbin/vgdisplay -c'], stdout=subprocess.PIPE, shell=True, stdin=subprocess.PIPE,close_fds=True ).stdout.readlines()
-except:
-    sys.exit(1)
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='lvm')
+    return parser.parse_args()
 
-template = "servers.{0}.system.lvm.{1}.{2} {3} {4}"
-hostname = gethostname().replace('.','_')
-timestamp = str(int(time.time()))
 
-if vgdisplay:
+def main():
+    args = parse_args()
+    vgdisplay = check_output(('/sbin/vgdisplay', '-c')).stdout.readlines()
```

Comment:
IDE says: Unresolved attribute reference 'stdout' for class 'str'
I guess the `stdout` has to be somewhere inside the `check_output` call, right?